packages/jaeger-ui/src/utils/readJsonFile.tsx

```diff
@@ -21,7 +23,18 @@ export default function readJsonFile(fileList: { file: File }): Promise<string>
         return;
       }
       try {
-        resolve(JSON.parse(reader.result));
+        const traceObj = JSON.parse(reader.result);
+        if ('resourceSpans' in traceObj) {
+          JaegerAPI.transformOTLP(traceObj)
```

Comment:
you previously wrote that transformOTLP action will be called automatically anyway. Does it mean it will be called twice now?